rules/why-upgrade-to-latest-dotnet/rule.md

```diff
@@ -0,0 +1,49 @@
+---
+type: rule
+archivedreason: 
+title: Do you know why to upgrade from .NET Framework to the latest .NET?
+guid: 75e296c8-1c21-4a4e-a2fb-4df32fd6140e
+uri: why-upgrade-to-latest-dotnet
+created: 2023-10-06T23:18:38.515Z
+authors:
+  - title: Yazhi Chen
+    url: https://www.ssw.com.au/people/yazhi-chen
+
+---
+In the world of software development, staying up-to-date with technology is not just a trend; it's a necessity. As the digital landscape evolves, so do the tools and frameworks that developers rely on to build applications. One significant transition that has been taking place in recent years is the migration from the legacy .NET Framework to the latest .NET (currently .NET 7 as of November 2022 and .NET 8 will be released on November 2023) offerings. But why is this shift so crucial, and what benefits does it bring to the table?
+<!--endintro-->
+
+## The Evolution of .NET
+To understand the significance of upgrading from .NET Framework to the latest .NET, let's first take a brief look at the evolution of the .NET platform.
+
+### .NET Framework: The Legacy
+
+The .NET Framework has been the backbone of Windows application development for nearly two decades. It has provided a robust and versatile environment for building Windows desktop applications, web applications, and services. However, as technology progressed, the limitations of .NET Framework became more apparent.
+
+#### The Pain Point:
+
+* Compatibility: .NET Framework applications primarily run on Windows, limiting cross-platform deployment.
```

Comment:
Change colons to dashes, it's easier to notice the split and aligns with the prefixes rules 

Take a look at https://www.ssw.com.au/rules/prefixes/